lib/direncrypt/consistency.py

```diff
@@ -41,7 +41,10 @@ def __init__(self, database):
         self.database = database
         with Inventory(self.database) as inventory:
             self.parameters = inventory.read_parameters()
-            self.fileset = inventory.read_register()
+            self.all_register = inventory.read_all_register()
```

Comment:
`all_register`, `registered_links` and `registered_dirs` are not used and should be removed.

I believe that this context with Inventory should better be moved to `check()`, but that can be done at another time.

Reply:
`all_register`, r`egistered_links` and `registered_dirs` removed. `self.registered_files` moved to `check()` method.